### PR TITLE
Streamline "The Future" text

### PR DESCRIPTION
**Pull request by eLtMosen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,21 +129,21 @@ eLtMosen has created the minimalistic [pulsedot watchface](https://www.youtube.c
...
 As we already mentioned in a [previous blog post](https://asteroidos.org/news/farewell-freenode), we moved all our communication from freenode to Matrix and Libera.chat.
 
 ## The Future
 
-As you might have noticed, the current releases on the installation page already seem to have a feature parity with the 2.0 release. The reason behind this is that at some point we decided to switch from our stable 1.0 release to the 1.1 nightly release as the 1.0 release became too old. In the future we would like to change our release cycle to release more often. So that a stable release will always be stable but not too old where it is no longer maintainable.
+As you might have noticed, the current releases linked on the installation pages, have feature parity with the 2.0 release. At some point, we decided to switch from our stable 1.0 release to a quasi 1.1 nightly rolling release, as the 1.0 release became too old to maintain. In the future, we would like to change our release cycle to offer more frequent stable releases. A stable release will always be stable. But not too old to no longer be maintainable.
```
Comment (FlorentRevest) [inline README.md:137]: There's an extra comma after "pages" in the first sentence

File: README.md [hunk 2]
```diff
@@ -129,21 +129,21 @@ eLtMosen has created the minimalistic [pulsedot watchface](https://www.youtube.c
...
-
-For the future we also would like to setup a roadmap to what we would like to see in an eventual next release.\
+For the future, we are going to set up a roadmap for features we would like to see in an eventual next release.\
 Based on recent early community work, we might see features like:
 
    * Combined fitness app (Privacy minded heart rate monitoring and step counter tracking)
    * WiFi setup via the settings app
    * Web based Watchface creation tool
-   * App store making community contributions easily available
+   * Web based flash tool
```
Comment (FlorentRevest) [inline README.md:145]: Mhh I have the feeling it isn't really ready to show though ?

File: README.md [hunk 3]
```diff
@@ -129,21 +129,21 @@ eLtMosen has created the minimalistic [pulsedot watchface](https://www.youtube.c
...
 
 ## The Future
 
-As you might have noticed, the current releases on the installation page already seem to have a feature parity with the 2.0 release. The reason behind this is that at some point we decided to switch from our stable 1.0 release to the 1.1 nightly release as the 1.0 release became too old. In the future we would like to change our release cycle to release more often. So that a stable release will always be stable but not too old where it is no longer maintainable.
+As you might have noticed, the current releases linked on the installation pages, have feature parity with the 2.0 release. At some point, we decided to switch from our stable 1.0 release to a quasi 1.1 nightly rolling release, as the 1.0 release became too old to maintain. In the future, we would like to change our release cycle to offer more frequent stable releases. A stable release will always be stable. But not too old to no longer be maintainable.
 
-
-For the future we also would like to setup a roadmap to what we would like to see in an eventual next release.\
+For the future, we are going to set up a roadmap for features we would like to see in an eventual next release.\
```
Comment (FlorentRevest) [inline README.md:139]: What's this \ at the end ? It was already there but maybe worth addressing ?

File: README.md (hunk 2 above)
Comment (eLtMosen) [reply README.md:145]: Same could be said about the web based watchface tool.
The section is lead in with "Based on recent early community work, we might see features like.". Imo its a teaser section for what could be happening based on current activities. And the "might" does not exclude "might not"?
I think in that context, we might even add berosets maliit work with "Virtual onscreen keyboard"? :P

Comment (FlorentRevest) [reply README.md:145]: Oh I am sorry I didn't read the section name and didn't realize this was in the prospective section... Nevermind! Thanks for pushing back :)

Comment (FlorentRevest) [reply README.md:145]: As for the keyboard... Why not, but this could be done in a separate PR, I'll merge this one already :)

**Pull request merged**